NDTensors/src/lib/TypeParameterAccessors/src/default_parameter.jl

```diff
@@ -0,0 +1,6 @@
+"""
+Get the default parameter of an object which is associated with a specific function tag.
+"""
+default_parameter(object, fun::typeof(Function)) = default_parameter(typeof(object), fun)
```

Comment:
```suggestion
default_parameter(object, fun::Function) = default_parameter(typeof(object), fun)
```
I think that's what you meant.

Reply:
Yes I think so, I fixed this. Thanks!